### ISSUE TITLE
Add to Atom core?

### ISSUE THREAD

**Issue by mnquintana** [atom/toggle-quotes#15]
I noticed this isn't installed in [Atom core](https://github.com/atom/atom/blob/master/package.json) – are there plans to add it soon? This seems like something that should be a feature shipped with Atom out of the box IMO.

**Comment by kevinsawicki** [comment]
Yeah, this wasn't included initially since it was still relatively new, but probably should be included by default now that it has matured.
